### ISSUE TITLE
How to skip decoding the specific field

### ISSUE THREAD

**Issue by BOFA1ex** [GoodforGod/DummyMapper#30]
```java
public class Subject {
    private String id;
    private LocalDateTime createTime;
}
```
```json
{
  "id" : "cd491a3f-317d-4e4d-aaef-662a85d7c33e",
  "createTime" : {
    "nano" : 0,
    "year" : 2050,
    "monthValue" : 5,
    "dayOfMonth" : 6,
    "hour" : 11,
    "minute" : 45,
    "second" : 3,
    "dayOfYear" : 126,
    "dayOfWeek" : "FRIDAY",
    "month" : "MAY",
    "chronology" : {
      "calendarType" : "iso8601",
      "id" : "ISO"
    }
  }
}
```
the `createTime` field should not be decoded expectly.
it should looks like `"createTime": 2020-12-18T14:57:00` 

can u support a annotation to specific field to skip it?
or append the `LocalDateTime/LocalDate/LocalTime` in mapping list which should be skipped.

**Comment by GoodforGod** [comment]
They actually should be skipped and properly serialized as ISO8601 dateTime, I'll look into it, thanks!

**Comment by GoodforGod** [comment]
Reproduced the issue, will look into it

**Comment by GoodforGod** [comment]
Reproduced issue for all of java.time.* package and fixed proper serialization for them is ISO8601 format

```
{
  "time" : "15:03:44",
  "date" : "1986-05-23",
  "dateTime" : "2077-10-07T02:59:01",
  "timestamp" : "2034-08-12T10:21:49.000+03:00",
  "dateOld" : "1970-01-11T16:08:32.308+03:00",
  "dateOldCoverage" : "1970-01-24T19:57:31.453+03:00",
  "dateTimeString" : "2029-01-17T00:25:13",
  "dateTimeObject" : "2026-12-08T21:10:44"
}
```

This fix will be available in 1.0.1 release soon

**Comment by GoodforGod** [comment]
Also @BOFA1ex you can skip field serialization by adding @JsonIgnore (jackson) annotation to field\getter
